Userland/Utilities/mkdir.cpp

```diff
@@ -8,27 +8,35 @@
 #include <AK/LexicalPath.h>
 #include <AK/StringBuilder.h>
 #include <LibCore/ArgsParser.h>
+#include <LibCore/System.h>
+#include <LibMain/Main.h>
 #include <errno.h>
 #include <sys/stat.h>
 #include <unistd.h>
 
-int main(int argc, char** argv)
+ErrorOr<int> serenity_main(Main::Arguments arguments)
 {
-    if (pledge("stdio cpath rpath", nullptr) < 0) {
-        perror("pledge");
-        return 1;
-    }
+    TRY(Core::System::pledge("stdio cpath rpath", nullptr));
```

Comment:
The `exec_pledges` argument can be omitted with the `System::pledge` wrapper.

```suggestion
    TRY(Core::System::pledge("stdio cpath rpath"));
```